astropy/io/ascii/tests/common.py

```diff
@@ -26,12 +26,15 @@ def teardown_function(function):
 
 # Compatibility functions to convert from nose to pytest
```

Comment:
Hmm, really these should just be replaced! But can be for follow-up...

Reply:
Right. I can do that as a follow up !

Reply:
There's an open issue for this already: #15782

Reply:
These are being removed already in #15839, but in most instances the corresponding `numpy.testing` functions would be used; so if I understood you correctly, the numpy versions would already take care of setting `__tracebackhide__`? A couple uses of `assert_equal` would be changed to simple `assert`s though.

Reply:
thanks @eerovaher , I wasn't aware of this issue. Since it's mark with "good first issue" I should leave it to new contributors, right ?

> if I understood you correctly, the numpy versions would already take care of setting `__tracebackhide__`?

Correct ! For instance, see the first line in `numpy.testing.assert_equal`'s definition https://github.com/numpy/numpy/blob/e920ca42d98c4911c2294784f8809b38735923a4/numpy/testing/_private/utils.py#L313

Reply:
Simple and non-critical tasks such as removing these functions are indeed best left for new contributors.

Reply:
> thanks @eerovaher , I wasn't aware of this issue. Since it's mark with "good first issue" I should leave it to new contributors, right ?
> 
It is all but done in #15839 already, but your comments would be very welcome there. Pretty much all that remains to be done is settling on where to best use `numpy.testing.assert_equal()` vs. `assert a == b`. And if this PR is being merged first, might be worth looking if the new `assert`s in #15839 should get their `__tracebackhide__` as well.

Reply:
Thank you, I'll make sure to review this one tomorrow !